packages/commonwealth/client/scripts/views/components/PreferenceTags/usePreferenceTags.tsx

```diff
@@ -0,0 +1,31 @@
+import { useState } from 'react';
+import { preferenceTags } from './mockedTags';
+import { PreferenceTagsHookProps, SelectedTag, Tag } from './types';
+
+const usePreferenceTags = ({
+  initialSelectedTag = [],
+}: PreferenceTagsHookProps) => {
+  const [selectedTags, setSelectedTags] = useState<SelectedTag[]>(
+    [...preferenceTags].map((item) => ({
+      item,
+      isSelected: !!initialSelectedTag.find((x) => x.tag === item.tag),
+    })),
+  );
+
+  console.log('selectedTags => ', selectedTags);
```

Comment:
lets remove this as later on it dangles around

Reply:
done, thanks for the spot.